prep-css-flexbox/display-flex.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - display: flex</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+    }
+
+    #card-deck {
+      padding: 20px;
+    }
```

Comment:
This should be combined into a single CSS ruleset:

```CSS
#card-deck {
  display: flex;
  padding: 20px;
}
```

Reply:
<img width="1296" alt="Screen Shot 2021-09-17 at 4 34 43 PM" src="https://user-images.githubusercontent.com/90414477/133865208-4d0c8c73-b2d1-4334-8ddd-1cfc144f7960.png">